crmsh/bootstrap.py

```diff
@@ -1757,7 +1757,6 @@ def join_ssh_merge(cluster_node, remote_user):
     known_hosts_new: set[str] = set()
 
     cat_cmd = "[ -e ~/.ssh/known_hosts ] && cat ~/.ssh/known_hosts || true"
-    #logger_utils.log_only_to_file("parallax.call {} : {}".format(hosts, cat_cmd))
```

Comment:
This is the chance to make lines unique before saving `known_hosts_new` to the disk. Currently, it easily gets duplicate lines.

Reply:
Please try again
I think this `duplicate` issue already gone